chore: multi-transaction ID references removal

saledjenic opened the pull request:
- Updated multiple services and tests to eliminate multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Eliminated multi-transaction ID from various structures and functions across the wallet service.
- Adjusted transaction management logic across the code base.
- Updated related queries and database interactions to reflect the removal of multi-transaction handling.
- Cleaned up related tests.

saledjenic committed:
chore: simplify transaction handling by removing multi-transaction ID

- Updated multiple services and tests to eliminate multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Adjusted transaction management logic across the code base.
- Cleaned up related tests.

saledjenic committed:
chore: remove multi-transaction ID references and clean up code

- Eliminated multi-transaction ID from various structures and functions across the wallet service.
- Updated related queries and database interactions to reflect the removal of multi-transaction handling.
- Cleaned up related tests.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (21)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 3b2cbe67 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/1/) | 2025-09-18 07:40:19 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-073705-3b2cbe-pr6939.zip) |
| :heavy_check_mark: | 3b2cbe67 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/1/) | 2025-09-18 07:47:45 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-073706-3b2cbe-pr6939.zip) |
| | | | | | | |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6939/2/) | 2025-09-18 07:40:35 | ~2 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-073736-704a7a-pr6939.zip) |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6939/2/) | 2025-09-18 07:41:36 | ~3 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-073741-704a7a-pr6939.zip) |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/2/) | 2025-09-18 07:43:34 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-074023-704a7a-pr6939.zip) |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/2/) | 2025-09-18 07:47:15 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/2//consoleText) |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/2/) | 2025-09-18 07:57:58 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-074749-704a7a-pr6939.zip) |
| :heavy_check_mark: | 704a7a8b | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/2/) | 2025-09-18 08:02:53 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/3/) | 2025-09-18 08:43:19 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083938-a6e659-pr6939.zip) |
| :heavy_check_mark: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6939/3/) | 2025-09-18 08:44:08 | ~4 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083942-a6e659-pr6939.zip) |
| :heavy_check_mark: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6939/3/) | 2025-09-18 08:44:47 | ~4 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083948-a6e659-pr6939.zip) |
| :heavy_check_mark: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/3/) | 2025-09-18 08:51:20 | ~11 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-083939-a6e659-pr6939.zip) |
| :heavy_multiplication_x: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/3/) | 2025-09-18 08:52:47 | ~13 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/3//consoleText) |
| :heavy_check_mark: | a6e65980 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/3/) | 2025-09-18 09:08:10 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/3//consoleText) |
| :heavy_check_mark: | a6e65980 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/4/) | 2025-09-18 09:18:48 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 08f0bcf6 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6939/4/) | 2025-09-18 09:33:28 | ~4 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092856-08f0bc-pr6939.zip) |
| :heavy_check_mark: | 08f0bcf6 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6939/4/) | 2025-09-18 09:33:38 | ~4 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092851-08f0bc-pr6939.zip) |
| :heavy_check_mark: | 08f0bcf6 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/4/) | 2025-09-18 09:40:30 | ~11 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092847-08f0bc-pr6939.zip) |
| :heavy_check_mark: | 08f0bcf6 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/4/) | 2025-09-18 09:48:08 | ~19 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-092848-08f0bc-pr6939.zip) |
| :heavy_check_mark: | 08f0bcf6 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/5/) | 2025-09-18 09:48:19 | ~19 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/5//consoleText) |
| :heavy_check_mark: | 08f0bcf6 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/4/) | 2025-09-18 10:05:29 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/4//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 3cf7c82e | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6939/5/) | 2025-09-18 13:24:17 | ~4 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131958-3cf7c8-pr6939.zip) |
| :heavy_check_mark: | 3cf7c82e | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6939/5/) | 2025-09-18 13:24:46 | ~4 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-132004-3cf7c8-pr6939.zip) |
| :heavy_check_mark: | 3cf7c82e | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/5/) | 2025-09-18 13:49:01 | ~29 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131954-3cf7c8-pr6939.zip) |
| :heavy_check_mark: | 3cf7c82e | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/5/) | 2025-09-18 13:57:52 | ~37 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-131956-3cf7c8-pr6939.zip) |
| :heavy_check_mark: | 3cf7c82e | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/6/) | 2025-09-18 14:00:29 | ~40 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/6//consoleText) |
| :heavy_check_mark: | 3cf7c82e | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/5/) | 2025-09-18 14:18:17 | ~58 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8dd6a15d | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6939/6/) | 2025-09-22 13:43:17 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-133954-8dd6a1-pr6939.zip) |
| :heavy_check_mark: | 8dd6a15d | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6939/6/) | 2025-09-22 13:43:24 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-133959-8dd6a1-pr6939.zip) |
| :heavy_check_mark: | 8dd6a15d | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6939/6/) | 2025-09-22 13:46:39 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-134004-8dd6a1-pr6939.zip) |
| :heavy_check_mark: | 8dd6a15d | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/7/) | 2025-09-22 13:50:02 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6939/7//consoleText) |
| :heavy_check_mark: | 8dd6a15d | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6939/6/) | 2025-09-22 13:50:55 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250922-133955-8dd6a1-pr6939.zip) |
| :heavy_multiplication_x: | 8dd6a15d | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/6/) | 2025-09-22 14:07:29 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6939/6//consoleText) |

saledjenic committed:
chore: remove unused methods from path processors

- Eliminated `Send` and `BuildTransaction` methods.
- Updated related tests to reflect the removal of these methods.
- Codebase cleaned up accordingly.

dlipicar commented on services/wallet/activity/common/types.go:
```diff
@@ -90,29 +90,28 @@ type Token struct {
+	PayloadType               PayloadType            `json:"payloadType"`
+	Key                       string                 `json:"key"`
+	Transaction               *TransactionIdentity   `json:"transaction,omitempty"`
+	Transactions              []*TransactionIdentity `json:"transactions,omitempty"`
```
I guess `Transactions` is no longer used? It was only there for the MultiTransaction case if I remember correctly
